packages/vscode-extension/src/extension.ts

```diff
@@ -213,36 +214,45 @@ export async function activate(context: vscode.ExtensionContext) {
 
   const openAccountLinkCmd = vscode.commands.registerCommand(
     "fx-extension.openAccountLink",
-    (...args) => Correlator.run(handlers.openAccountLinkHandler, args)
+    (...args) =>
+      Correlator.run(handlers.openAccountLinkHandler, [TelemetryTriggerFrom.ViewTitleNavigation])
   );
   context.subscriptions.push(openAccountLinkCmd);
 
   const createAccountCmd = vscode.commands.registerCommand(
     "fx-extension.createAccount",
-    (...args) => Correlator.run(handlers.createAccountHandler, args)
+    (...args) =>
+      Correlator.run(handlers.createAccountHandler, [TelemetryTriggerFrom.ViewTitleNavigation])
```

Comment:
What about also distinct the docs content(which tree section it's from)?

Reply:
Fixed by adding a new property `documentation-name`. Thank you!